packages/shared/src/components/sidebar/SidebarBottomSection.tsx

```diff
@@ -41,7 +41,13 @@ export function SidebarBottomSection({
       icon: () => <ListIcon Icon={() => <TerminalIcon />} />,
       title: 'Changelog',
       path: `${process.env.NEXT_PUBLIC_WEBAPP_URL}sources/daily_updates`,
-      navItemRef,
+      alert: changelog.isAvailable && (
+        <AlertDot
+          className="right-5 mt-1.5"
+          color={AlertColor.Cabbage}
+          position={AlertDotPosition.Fixed}
```

Comment:
Should probably still work with absolute right, as it's absolute based on the sidebar element?

Reply:
It is absolute based on the Icon element within the sidebar element, that's why keeping it absolute wouldn't work and had to add this.

Reply:
Ah it should be fixed to the changelog element alone, I see now it works in relation to the whole sidebar.

Reply:
I think something like this might work:
```
      rightIcon: (_) =>
        changelog.isAvailable && (
          <AlertDot className="-top-1 right-0" color={AlertColor.Cabbage} />
        ),
```

Would look like this:
<img width="651" alt="Screenshot 2023-10-14 at 09 46 07" src="https://github.com/dailydotdev/apps/assets/554874/ebccdd8c-758b-4e94-82ed-da6ffa37aef0">

Reply:
Ah yes, I hadn't explored the right icon prop. Thank you! 🙌 Will keep this and remove the additions to AlertDot.